frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/EvidenceModal.js

```diff
@@ -474,17 +474,17 @@ const EvidenceModal = ({ caseData, documentSubmission = [], setShow, userRoles,
       case "RE_SCHEDULE":
         return type === "reject" ? "REJECTION_RESCHEDULE_REQUEST" : "INITIATING_RESCHEDULING_OF_HEARING_DATE";
       case "WITHDRAWAL":
-        return type === "reject" ? "REJECT_VOLUNTARY_SUBMISSIONS" : "WITHDRAWAL";
+        return "WITHDRAWAL";
```

Comment:
removing type === reject check, is this intentional ?

Reply:
Yes, it is necessary, we will be having the same order type with Approved/Rejected in it